x-pack/plugins/cloud_security_posture/server/plugin.test.ts

```diff
@@ -222,7 +222,7 @@ describe('Cloud Security Posture Plugin', () => {
       expect(spy).toHaveBeenCalledTimes(0);
     });
 
-    it('packagePolicyPostCreate should return the updated packagePolicy', async () => {
+    it('packagePolicyPostCreate should return the same received policy', async () => {
```

Comment:
could you clarify this change? is it because we don't write to the package policy on updates/init ?

Reply:
> could you clarify this change? is it because we don't write to the package policy on updates/init ?

Yeah, since we have removed the post hook that populates the `data.yaml` with the relevant rules, we expect the post create action to not touch the policy at all